devops/denv/Dockerfile

```diff
@@ -1,89 +1,87 @@
-FROM steptan/ubuntu-1604-rockdb-5.13.3-sharedlib 
+FROM sovrin/python3:0.2.0-focal
 
 ARG INDY_NODE_VERSION
 ARG POOL_IP
 
 USER root
 
-RUN apt-get update -y && apt-get install -y \
-    apt-transport-https \
-    ca-certificates
-
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys BD33704C
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys F23C5A6CF475977595C89F51BA6932366A755776
-RUN echo "deb http://us.archive.ubuntu.com/ubuntu xenial main universe" >> /etc/apt/sources.list
-RUN echo "deb https://repo.sovrin.org/deb xenial master" >> /etc/apt/sources.list
-RUN echo "deb https://repo.sovrin.org/sdk/deb xenial master" >> /etc/apt/sources.list
-RUN echo "deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu xenial main" >> /etc/apt/sources.list
-
-RUN apt-get update -y && apt-get install -y \
-    # common stuff
-    git \
-    wget \
-    unzip \
-    python3 \
-    python3-pip \
-    python-setuptools \
-    python3-venv \
-    vim \
-    sudo \
-    ursa \
-    # fpm
-    ruby \
-    ruby-dev \
-    rubygems \
-    gcc \
-    make \
-    # rocksdb python wrapper
-    libgflags-dev \
-    libbz2-dev \
-    libzstd-dev \
-    zlib1g-dev \
-    liblz4-dev \
-    libsnappy-dev \
-    # libindy
-    python3-nacl \
-    libindy-crypto=0.4.5 \
-    libindy \
-    indy-cli \
-    libsovtoken=1.0.2~92
-#//python3-indy-crypto
-
-RUN pip3 install -U \
-    setuptools==50.3.2 \
-    pytest==4.6.2 \
-    zipp==1.0.0 \
-    importlib_metadata==2.1.1 \
-    pytest-xdist==2.2.1 \
-    python3-indy==1.16.0 \
-    mock==3.0.5
-
 RUN useradd -m -d /home/indy -s /bin/bash -p $(openssl passwd -1 "token") -u 1000 indy; usermod -aG sudo indy
-RUN ln -s /usr/bin/pip3 /usr/bin/pip
 
 WORKDIR /home/indy
 
 # indy-node along with supervisor
 ENV INDY_NODE_VERSION ${INDY_NODE_VERSION:-1.13.0~dev1225}
 ARG indy_plenum_ver=1.13.0~dev1033
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-RUN echo "deb https://repo.sovrin.org/deb xenial rc" >> /etc/apt/sources.list \
-    && apt-get update && apt-get install -y \
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88 \
+    && echo "deb https://repo.sovrin.org/sdk/deb xenial master" >> /etc/apt/sources.list \
+    && echo "deb https://repo.sovrin.org/deb xenial stable master" >> /etc/apt/sources.list \
+    && echo "deb http://archive.ubuntu.com/ubuntu/ xenial universe main" >> /etc/apt/sources.list \
+    && apt-get update && apt-get install -y --allow-downgrades \
+    python3-jsonpickle=0.9.6 \
```

Comment:
had to pin a lot of dependencies required by python3.5 version of plenum to work with 20.04